mlir/test/Dialect/Rock/lowering_output_swizzle.mlir

```diff
@@ -0,0 +1,60 @@
+// RUN: rocmlir-opt --rock-output-swizzle %s | FileCheck %s
```

Comment:
I d like to see another test that runs passes upto -rock-threadwise-gemm-lowering which lowers `threadwise_write_all` where we can see the vectorization has actually changed. Will that be a possible test to add ?